### PR TITLE
Feature/UI home segments

### LINKED ISSUE #773: UI Feature: Clean named Segment from Homepage
If the `MapSegmentationCapability` capability is present, we can fetch all Segments from `/api/v2/robot/capabilities/MapSegmentationCapability`, filter for those which are named and build a multi-select capable list of them just like with the ValetudoZonePresets:

image

### PR DESCRIPTION
**Pull request by Jomik**
## Type of change
UI implementation of capability

# Description 
Shows a dialog, similar to Zones, to allow you to select named segments, and clean them.
Implemented using "source-code copy/paste", as refactoring this with all the global variables is a pain.

Fixes #773

### CODE REVIEW

File: lib/robots/mock/capabilities/MockMapSegmentationCapability.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ */
+class MockMapSegmentationCapability extends MapSegmentationCapability {
+    /**
+     * @returns {Promise<Array<import("../../../entities/core/ValetudoMapSegment")>>}
+     */
+    async getSegments() {
+        return [
+            new ValetudoMapSegment({ id: "foo", name: "Foo"}),
+            new ValetudoMapSegment({ id: "boo", name: "Bar"}),
+            new ValetudoMapSegment({ id: "empty" })
```
Comment (SBeyeMHP) [inline lib/robots/mock/capabilities/MockMapSegmentationCapability.js:15]: Uh? Empty?

**Review by SBeyeMHP — CHANGES_REQUESTED** [review]

File: lib/robots/mock/capabilities/MockMapSegmentationCapability.js (hunk 1 above)
Comment (Hypfer) [inline lib/robots/mock/capabilities/MockMapSegmentationCapability.js:15]: Why does the backend return the "empty" label?

**Review by Hypfer — CHANGES_REQUESTED** [review]

File: lib/robots/mock/capabilities/MockMapSegmentationCapability.js (hunk 1 above)
Comment (Jomik) [reply lib/robots/mock/capabilities/MockMapSegmentationCapability.js:15]: I guess I could have written something else. It is just an id for the segment with an empty/undefined name.

Comment (Jomik) [reply lib/robots/mock/capabilities/MockMapSegmentationCapability.js]: I changed the IDs to avoid confusion.

**Pull request merged**